rebar.config

```diff
@@ -1,4 +1,6 @@
-{erl_opts, [debug_info]}.
+{erl_opts, [debug_info,
+           {platform_define, "^2", 'RAND'}
```

Comment:
This could be problematic in Erlang/OTP 30 ;)

Reply:
So what is your suggestion?
I will check to how to fix it later.

Reply:
Actually, as long as we are doing this, `crypto:strong_rand_bytes/1` has been around since [OTP_R14B04](https://github.com/erlang/otp/blob/OTP_R14B04/lib/crypto/src/crypto.erl#L50). I think you can delete ALL of this code and just use `strong_rand_bytes/1` in the code where you're checking this. It should be safe to let the thing crash on `low_entropy` errors. I don't think that should be a realistic risk on any modern-ish Erlang, since the code won't even get run (`ssl:seed/1` hasn't been used in OTP since [R14](https://github.com/erlang/otp/blob/OTP_R14B/lib/ssl/src/ssl.erl#L31)) on any supported system.

We could even take out the entire case statement looking to see if `ssl:seed/1` is exported. I really don't see the point in October 2018. I don't think anyone on OTP R14 is keeping lhttpc up to date. Any thoughts @motobob?

Reply:
erlcloud which uses this is R16+ and frankly should be already made R19.3+ so let' remove all this